pkg/controller/data_upload_controller.go

```diff
@@ -98,7 +99,12 @@ func NewDataUploadReconciler(
 	preparingTimeout time.Duration,
 	log logrus.FieldLogger,
 	metrics *metrics.ServerMetrics,
+	dataMovePriorityClass string,
 ) *DataUploadReconciler {
+	if dataMovePriorityClass != "" {
```

Comment:
It doesn't look like necessary as we have already printed the same log in Velero server startup or node-agent server startup.